content/400-reference/300-database-reference/01-database-features.mdx

```diff
@@ -56,11 +56,11 @@ This section describes which database features exist on the relational databases
 
 ### Indexes
 
-| Index    | PostgreSQL | Microsoft SQL Server | MySQL | SQLite | CockroachDB | Prisma schema                                                                                     | Prisma Client | Prisma Migrate |
-| -------- | ---------- | -------------------- | ----- | ------ | ----------- | ------------------------------------------------------------------------------------------------- | ------------- | -------------- |
-| `UNIQUE` | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | [`@unique` and `@@unique`](/concepts/components/prisma-schema/data-model#defining-a-unique-field) | ✔️            | ✔️             |
-| `WHERE`  | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | Not yet                                                                                           | ✔️            | Not yet        |
-| `USING`  | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | ✔️†                                                                                               | ✔️            | Not yet        |
+| Index    | PostgreSQL | Microsoft SQL Server | MySQL | SQLite | CockroachDB | Prisma schema                                                                                                                | Prisma Client | Prisma Migrate |
+| -------- | ---------- | -------------------- | ----- | ------ | ----------- | ---------------------------------------------------------------------------------------------------------------------------- | ------------- | -------------- |
+| `UNIQUE` | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | [`@unique` and `@@unique`](/concepts/components/prisma-schema/data-model#defining-a-unique-field)                            | ✔️            | ✔️             |
+| `WHERE`  | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | Not yet                                                                                                                      | ✔️            | Not yet        |
+| `USING`  | ✔️         | ✔️                   | ✔️    | ✔️     | ✔️          | [`type`](/concepts/components/prisma-schema/indexes#configuring-the-access-type-of-indexes-with-type-postgresql)<sup>†</sup> | ✔️            | Not yet        |
 
 - † Available with the PostgreSQL connector only in Prisma versions `4.0.0` and later.
```

Comment:
Semi (un)related: Is this version correct?

Reply:
It's correct if we're talking about GA, but not preview. So potentially better to have:

```suggestion
- † Available in preview in 3.6.0 and later and in general availability in 4.0.0 and later, with the PostgreSQL connector only.
```